test/deployframework/context.go

```diff
@@ -69,6 +69,14 @@ func (df *DeployFramework) NewDeployerCtx(
 		},
 	}
 
+	// validate the reporting-operator image isn't empty when overrided
+	if df.ReportingOperatorImageRepo != "" || df.ReportingOperatorImageTag != "" {
```

Comment:
Should we do the same for meteringOperatorImageRepo and meteringOperatorImageTag ?

Reply:
The metering operator images get a default value (defined in the Makefile) so I didn't think we would need to validate this image, but I can add a check that verifies both the image repo and tag are both set, which should be sufficient.